### PR TITLE
[TECH-506] Ship typed file with mpyl

### PR DESCRIPTION
**Pull request by koeves**
branch: feature/TECH-506_ship_typing_info

----
### 📕 [TECH-506](https://vandebron.atlassian.net/browse/TECH-506) Ship typing information with mpyl library danielkoves@vandebron.nl 
See [180](https://github.com/Vandebron/mpyl/issues/180) 

🏗️ Build [2](https://jenkins.k8s-serv-backend.vdbinfra.nl/job/MPyL%20Pipeline%20-%20Test/job/PR-341/2/display/redirect) ✅ Successful, started by _Jorg Post_  
🚀 *[cloudfront-service](https://cloudfront-service-341.test.nl/swagger/index.html)*, *example-dagster-user-code*, *job*, *kong-sync*, *[nodeservice](https://nodeservice-341.test.nl/swagger/index.html)*, *ocpp-first*, *ocpp-second*, *[sbtservice](https://sbtservice-341.test.nl/swagger/index.html)*, *sparkJob-first*, *sparkJob-second*  

[TECH-506]: https://vandebron.atlassian.net/browse/TECH-506?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -61,4 +61,5 @@ def get_install_requirements():
     include_package_data=True,
     packages=setuptools.find_packages(where="./src"),
     python_requires=">= 3.9",
+    package_data={"mpyl": ["py.typed"]},
```
Comment (Jorg88) [inline setup.py:64]: maybe you can assign `setuptools.find_packages(where="./src")` to a variable and use that both here and in `packages`?

Comment (koeves) [reply setup.py:64]: no, we just need to include the py.typed file here

Comment (Jorg88) [reply setup.py:64]: I'm referring to the key `"mpyl"`, in the docs of mypy those seem to be the same but I don't actually know what `setuptools.find_packages(where="./src")` returns. 

```
    package_data={"package_b": ["py.typed", "lib.pyi"]},
    packages=["package_b"]
```

Comment (koeves) [reply setup.py:64]: I think these are two different things. `packages` is the list of packages in the whole mpyl library. `package_data` is a mapping from a package to extra files it should include.

**Review by Jorg88 — APPROVED** [review]

**Review by SamTheisens — APPROVED** [review]
Nice!

**Comment by Jorg88** [comment]
Release notes btw!

**Pull request merged**